src/it/scala/com/bwsw/cloudstack/vault/server/e2e/NonExistentEntityEventHandlingTestSuite.scala

```diff
@@ -116,4 +122,21 @@ class NonExistentEntityEventHandlingTestSuite extends FlatSpec with Checks with
   override def afterAll(): Unit = {
     components.close()
   }
+
+  private def waitAccountAndVmDeletion(accountId: UUID, vmId: UUID): Unit = {
```

Comment:
maxRetryCount should be a method parameter

Reply:
ok